app/Bootstrap.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * @author Jakub Vyvazil <jakub@vyvazil.cz> (https://vyvazil.cz)
```

Comment:
This annotation is useless these days. It's already in Git. :-)

Reply:
Thanks, I was using the syntax from the other files. Fixed.